src/event.hxx

```diff
@@ -267,6 +270,25 @@ public:
 	EventGridSelectNewChosen(int t = -1, int s = -1):track(t),scene(s) {}
 };
 
+class EventLooperBarsToRecord : public EventBase
```

Comment:
Since the information how many Bars should be recorded belongs to the clip, not the looper (which is the track) I would suggest to rename this to "EventClipBarsToRecord"